modules/Bio/EnsEMBL/Variation/DBSQL/VariationFeatureAdaptor.pm

```diff
@@ -1773,6 +1773,11 @@ sub fetch_by_hgvs_notation {
     $reference =~ s/\.\d+//g;
    }
 
+  # Old code only worked with right transcript version
+  # Delete transcript version to avoid problems
+  my $reference_no_version = $reference;
+  $reference_no_version =~ s/\.[0-9]+//;
```

Comment:
It would be better to keep the version in the sequence look up if possible, so we return exacly what was requested.